### ISSUE TITLE
Needs an update

### ISSUE THREAD

**Issue by yarons** [ClaudioSobrinho/Whatsapp-rambox-dark-theme#1]
Doesn't work with the current WhatsApp web.

**Comment by ClaudioSobrinho** [comment]
You're right, it stopped working a couple of versions ago. But now WhatsApp is releasing dark mode for all their platforms so I'm waiting to see if it still makes sense to invest time in this

**Comment by yarons** [comment]
I see, thank you!
